src/groups/mwc/mwcu/mwcu_blobiterator.t.cpp

```diff
@@ -274,10 +273,8 @@ static void test3_forwardIterator()
             ASSERT_NE_D("line " << data.d_line, iter2, end);
             const mwcu::BlobPosition p(iter2.position());
             const mwcu::BlobIterator it(iter2++);
-            bsl::iterator_traits<mwcu::BlobIterator>::reference r(*it);
```

Comment:
Removed in cpp20 anyway
https://en.cppreference.com/w/cpp/iterator/iterator_traits